jcliff.spec

```diff
@@ -0,0 +1,74 @@
+Name: 		      jcliff
+Version: 	      2.12.6
+Release: 	      1%{?dist}
+Summary: 	      JBoss configuration client front-end
+License: 	      2013, Red Hat, Inc. and/or its affiliates.
+Group: 		      Applications/File
+URL:           	https://github.com/bserdar/jcliff
+Source0:       	https://github.com/bserdar/jcliff/releases/download/v%{version}/%{name}-%{version}-dist.tar.gz
+BuildArch:      noarch
+BuildRoot: 	    %{_tmppath}/%{name}-%{namedversion}-%{release}-root
+BuildRequires:  unzip
+
+%description
+Jcliff configures a running instance of EAP6/JBoss7 using modular configuration files.
+
+%prep
+tar -xf %{_sourcedir}/%{name}-%{version}-dist.tar.gz -C %{_sourcedir}
+
+%install
+mkdir -p %{buildroot}/%{_datadir}/jcliff
+mkdir -p %{buildroot}/%{_bindir}/jcliff
+mv %{_sourcedir}/%{name}-%{version}/* %{buildroot}/%{_datadir}/jcliff
+ls %{buildroot}/%{_datadir}/jcliff
+ln -s %{_datadir}/%{name}-%{version} %{buildroot}%{_datadir}/jcliff
+ln -s %{_datadir}/%{name}-%{version}/jcliff %{buildroot}%{_bindir}/jcliff
+
+%files
+%defattr(0644,root,root,0755)
+%{_bindir}/jcliff/jcliff
+%{_datadir}/jcliff/COPYING
+%{_datadir}/jcliff/README.markdown
+%{_datadir}/jcliff/cookcc-0.3.3.jar
+%{_datadir}/jcliff/jboss-dmr-1.1.1.Final.jar
+%{_datadir}/jcliff/jcliff
+%{_datadir}/jcliff/jcliff-2.12.6
```

Comment:
@hcherukuri Can we generate this part ? Or move all the jars into a deps/ folder and simply include deps/ ? I don't like the idea of editing this SPEC file every time a dependency changes.

Reply:
If we add all the jars in the directory, I checked if we add just the directory then it should work.